lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -462,6 +476,28 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
                         />
                       </>
                     )}
+                    {promptForGradable && (
+                      <>
+                        <div className="sm:col-span-2 border-b" />
+                        <PanelLabel isCurrentStep verticalAlign="center">
+                          Max Points
```

Comment:
Even though that's how it looks in most of our designs, we decided to not capitalize every word 😅 

```suggestion
                          Max points
```